### DISCUSSION TITLE
The idea to add a pattern for an image file name

### DISCUSSION THREAD

**Discussion by ostap667inbox** [seruva19/kubin#67] [Ideas]
Now seed comes first in the image file name. However, it is a random number. Many picture viewers sort files by filename by default.

To sort in creation order, it would be nice to be able to have a date/time of creation in the beginning of the file name, something like datetime in "%Y%m%d%H%M%S" in Automatic1111. For example: 20230515152506-... at the beginning of the file allows you to correctly sort the list of images.
By default or as an option in the settings.

**Comment by seruva19** [comment]
The scheme you suggested is made default now ([468e113](https://github.com/seruva19/kubin/commit/468e1132d9eb88fea2d397ed426ef59809bd190d)), it's indeed more user-friendly. 
And later it will be possible to change it through UI.
